src/components/flat-table/flat-table-row/flat-table-row.style.ts

```diff
@@ -28,7 +28,7 @@ const firstColumnOldFocusStyling = `
 
 const newFocusStyling = `
   ${addFocusStyling(true)}
-  z-index: 1000;
+  z-index: 1005;
```

Comment:
suggestion (non-blocking): it might be better to use a more consistent approach like you have below in `flat-table.style`

```
z-index: ${({ theme }) => theme.zIndex.overlay + 5};

Reply:
This has been refactored to use the same approach as in `flat-table.style`